model-engine/model_engine_server/common/pydantic_types/endpoint_predict_payload.py

```diff
@@ -1,10 +0,0 @@
-from typing import Any, Optional
-
-from pydantic import BaseModel
-
-
-class EndpointPredictPayload(BaseModel):
```

Comment:
is this not used anywhere?

Reply:
I did a ctrl-f and did not see it anywhere. We'll see if tests fail..